features/support/hooks.rb

```diff
@@ -47,13 +47,10 @@ def capture_error(scenario, exception, js: false)
 
     # Using the feature (file) as the fingerprint means we only
     # get one issue per feature, keeping the dashboard simple.
-    # The hint is used by govuk_app_config, which will emit stats
-    # for the error class, in addition to what we get in Sentry.
     Sentry.capture_message(
       message,
       fingerprint: [scenario.location.file],
       backtrace: exception.backtrace,
-      hint: { exception: exception }
```

Comment:
Without this, we'll lose the [statsd error counting](https://github.com/alphagov/govuk_app_config/blob/87e445eccee5fba2449a170d5ba628e8a380fcb8/lib/govuk_app_config/govuk_error/configuration.rb#L107-L115), but I don't think we intended to use that anyway, so it's probably fine.

Can you add more detail about _why_ it fails? (The [commit where it's added](https://github.com/alphagov/smokey/pull/931/commits/a3c2836458edb537864f755d7fa3a9b1d46d68cd) references code in [sentry-ruby/lib/sentry/hub.rb](https://github.com/getsentry/sentry-ruby/blob/7848d59450ae6bcb8dea6c141abdcdfe7b218567/sentry-ruby/lib/sentry/hub.rb#L100) but it doesn't look like it's changed much since then).

Is there an error message or does it just fail to work? Can you link to a Sentry issue/error that proves this change works? etc

Also I'm not sure why we refer to `Sentry` directly, rather than `GovukError`, which is the [documented approach in govuk_app_config](https://docs.publishing.service.gov.uk/repos/govuk_app_config.html#error-reporting). Perhaps this wouldn't have broken if we'd have used that as an abstraction layer, since that's kept up to date? Could be worth looking into.

Reply:
So I'm not sure this has ever worked, the error message is the same as on the issue linked, not very helpful.

You can see the errors [here](https://govuk.sentry.io/issues/?environment=development&project=6370326&statsPeriod=7d) from my local run.

I tested calling GovukError instead of Sentry directly and the results are exactly the same, it works only without the hint. I also thought it was weird we're calling Sentry directly so I'll change it to use GovukError instead.